### PR TITLE
[release-2.4] trigger build to fix cve

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/stolostron/backlog/issues/22153
Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: .triggerbuild [hunk 1]
```diff
@@ -0,0 +1 @@
+Thu May  5 08:35:06 UTC 2022
```
Comment (elgnay) [inline .triggerbuild:1]: Should we use a more maintainable time format instead?

Comment (haoqing0110) [reply .triggerbuild:1]: @elgnay this value comes from `date > .triggerbuild`, do you want me to change the format like `2022-05-05`?

Comment (elgnay) [reply .triggerbuild]: yes, that looks better.

**Pull request merged**